actionpack/test/controller/log_subscriber_test.rb

```diff
@@ -92,6 +92,7 @@ class ACLogSubscriberTest < ActionController::TestCase
 
   def setup
     super
+    ActionView::Base.enable_fragment_cache_logging = true
```

Comment:
After we move the config into `ActionPack`, the config will look like this
```ruby
@controller.config.perform_caching = true
@controller.enable_fragment_cache_logging = true
```

And I think it's just the setup step in tests, like `@controller.config.perform_caching = true`